### DISCUSSION TITLE
Ferdium 6.3 on Windows - Ferdium not starting anymore

### DISCUSSION THREAD

**Discussion by francwalter** [ferdium/ferdium-app#1230] [Q&A]
Hello 
I have switched from Hamsket to Ferdium (because Hamsket was crashing all the time).
I installed Ferdium for all users (in Program Files).
Then, after maybe two months of use, Ferdium as well crashed often, used very much RAM despite of all services have Hibernate activated. 
Now it even wont start anymore. I see some Ferdium.exe processes in Task manager but no window.

I have about 10 services, some of them Custom Website (the main reason for apps like Hamsket or Ferdium, by the way).
I deleted the settings.json, a new one is created, no change, Ferdium wont be visible.
If I start the Portable (without any service) it starts, so it must be related to my services.
But how can I check?
Where is the error log?
I tried the 6.3.0, the 6.4.0-nightly-1 and 2, installing for me or for all users, always the same.
When I run the portable it works, but all my logins to the services are gone (after login to Ferdium server). 

I am on Windows 10 (64-Bit, Enterpr.) with 16 GB RAM (not enogh, nowadays, I have often crashes because of memory, Ferdium too, I guess).
Thanks for hints.
frank

**Comment by Alphrag** [comment]
Hi @francwalter, sorry to hear that.
The old setup was to install Ferdium only for the current user. With the change to allow for installation in Program Files maybe some files got corrupted when you made the switch. I had the same problem at some point (I couldn't figure out where this came from) but by looking at the files in the Ferdium folder (somewhere in `AppData` for me) I managed to find which `.json` file was corrupted (it was basically empty) and deleting it allowed Ferdium to restart. One way that can help figure out which file is the culprit is to start Ferdium from the cmd/powershell and use the error thrown to advance in your search.

We know that Ferdium is consuming a fair amount of RAM, which is due to how electron works and the fact that since we keep everything in containers it needs more memory than a browser which is optimised to share similar resources. 
We unfortunately are already struggling to keep Ferdium working and can't really tackle such a problem that would require a deep knowledge of electron and quite a rewrite (if at all possible with what we want to have).

As for the logins, this happens because Portable versions only use the folder that they create in the same place as where the executable is located, it is not using the partition folder that the executable installed globally uses. Even if you would copy over the content, this is unlikely to work directly because some service check hash would not match, which would force you to setup some services and login again on some others.
For many privacy and security reasons, we do not synchronise cookies login and others, which means that you need to login to all your services when using the Ferdium server. 

Hope this helps.
